src/platform/packages/shared/kbn-unified-tabs/src/components/tabbed_content/tabbed_content.tsx

```diff
@@ -23,6 +23,22 @@ import {
 } from '../../utils/manage_tabs';
 import type { TabItem, TabsServices, TabPreviewData } from '../../types';
 
+// TODO replace with real data when ready
+const RECENTLY_CLOSED_TABS_MOCK = [
+  {
+    label: 'Session 4',
+    id: '4',
+  },
+  {
+    label: 'Session 5',
+    id: '5',
+  },
+  {
+    label: 'Session 6',
+    id: '6',
+  },
+];
+
```

Comment:
Placed the mock here, not in examples, as my hunch is it'll be in state, is that correct? 
That said, I didn't want to alter the current state, as I believe you have the architecture of this planned @jughosta 

Let me know if we should modify this

Reply:
Yeah, I talked to Davis that it's not clear what "recent" means yet, that's also why I placed the mock here, as we're not sure how to get this data yet